src/test/scala/lms/utils.scala

```diff
@@ -10,7 +10,7 @@ trait LibSuite extends FunSuite {
 }
 
 trait TutorialFunSuite extends LibSuite {
-  val overwriteCheckFiles = false // should be false; temporary set to true only to simplify development
+  val overwriteCheckFiles = true // should be false; temporary set to true only to simplify development
```

Comment:
make sure this is `false` when you are ready to merge

Reply:
fixed in 3b22766